libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -311,7 +317,33 @@ string ABIFunctions::conversionFunction(Type const& _from, Type const& _to)
 			}
 			else if (toCategory == Type::Category::FixedPoint)
 			{
-				solUnimplemented("Not yet implemented - FixedPointType.");
+				if (fromCategory == Type::Category::Integer)
+				{
+					body =
+						Whiskers("converted := <cleanFixed>(mul(<cleanInt>(value), <factor>))")
+						("cleanInt", cleanupFunction(_from, false))
+						("cleanFixed", cleanupFunction(_to, false))
+						("factor", toString(pow(bigint(10), dynamic_cast<FixedPointType const &>(_to).fractionalDigits())))
+						.render();
+				}
+				else if (fromCategory == Type::Category::RationalNumber)
+				{
+					RationalNumberType const& fromRationalType = dynamic_cast<RationalNumberType const&>(_from);
```

Comment:
Can you move this to right after the `else if (toCategory == Type::Category::FixedPoint)`?